deflect/Event.h

```diff
@@ -60,6 +60,50 @@ namespace deflect
  * Typically used to forward user inputs from a window to classes that
  * generate content for it.
  *
+ * Events can be divided into several categories, listed below. Applications can
+ * decide to handle all the events or limit themselves to any subset that is
+ * appropriate.
+ *
+ * For example, an application which supports multiple touch points natively may
+ * want to use unprocessed touch points, while still assigning some specific
+ * actions to high-level gestures such as swipe events.
+ *
+ * For a concrete code example of basic event processing, users can refer to
+ * the simplestreamer demo application.
+ *
+ * Stream events
+ * - EVT_VIEW_SIZE_CHANGED   < remote stream window resized by user
+ * - EVT_CLOSE               < remote stream window closed by user
+ *
+ * Basic interaction
+ * - EVT_PRESS               < touch/mouse button press (single touch point)
+ * - EVT_MOVE                < touch point/mouse move (single touch point)
+ * - EVT_RELEASE             < touch/mouse button release (single touch point)
+ * - EVT_CLICK               < tap/click (with one or more fingers/buttons)
+ * - EVT_DOUBLECLICK         < double tap/click (with one or more fingers/buttons)
+ * - EVT_TAP_AND_HOLD        < tap and hold (with one or more fingers/buttons)
+ *
+ * Advanced gestures
+ * - EVT_PAN                 < pan with two or more finger
+ * - EVT_PINCH               < pinch with two finger
+ * - EVT_SWIPE_LEFT          < swipe left with two fingers
+ * - EVT_SWIPE_RIGHT         < swipe right with two fingers
+ * - EVT_SWIPE_UP            < swipe up with two fingers
+ * - EVT_SWIPE_DOWN          < swipe down with two fingers
+ *
+ * Unprocessed touch points
+ * - EVT_TOUCH_ADD           < touch point added (key field contains point id)
+ * - EVT_TOUCH_UPDATE        < touch point moved (key field contains point id)
+ * - EVT_TOUCH_REMOVE        < touch point removed (key field contains point id)
+ *
+ * Keybord events
+ * - EVT_KEY_PRESS           < key pressed
+ * - EVT_KEY_RELEASE         < key released
+ *
+ * Deprecated/unused:
+ * - EVT_NONE                < normally unused
+ * - EVT_WHEEL               < not sent by Tide >= 1.2, replaced by EVT_PINCH
```

Comment:
wheel is not deprecated imo, in a mouse-only deflect server it fits perfectly with the upper move, click, etc.